spec/service_spec.rb

```diff
@@ -109,38 +109,55 @@
   end
 
   context "#get_resources_by_uri_path" do
-    it "fetches data_center" do
-      return_message = <<-EOX.chomp
+    let(:return_message) do
+      <<-EOX.chomp
 <?xml version="1.0" encoding="UTF-8" standalone="yes"?>
 <data_center href="/api/datacenters/00000001-0001-0001-0001-0000000000f1" id="00000001-0001-0001-0001-0000000000f1">
-    <name>Default</name>
-    <description>The default Data Center</description>
-    <link href="/api/datacenters/00000001-0001-0001-0001-0000000000f1/storagedomains" rel="storagedomains"/>
-    <link href="/api/datacenters/00000001-0001-0001-0001-0000000000f1/clusters" rel="clusters"/>
-    <link href="/api/datacenters/00000001-0001-0001-0001-0000000000f1/networks" rel="networks"/>
-    <link href="/api/datacenters/00000001-0001-0001-0001-0000000000f1/permissions" rel="permissions"/>
-    <link href="/api/datacenters/00000001-0001-0001-0001-0000000000f1/quotas" rel="quotas"/>
-    <link href="/api/datacenters/00000001-0001-0001-0001-0000000000f1/qoss" rel="qoss"/>
-    <link href="/api/datacenters/00000001-0001-0001-0001-0000000000f1/iscsibonds" rel="iscsibonds"/>
-    <local>false</local>
-    <storage_format>v3</storage_format>
-    <version major="3" minor="6"/>
-    <supported_versions>
-        <version major="3" minor="6"/>
-    </supported_versions>
-    <status>
-        <state>up</state>
-    </status>
-    <mac_pool href="/api/macpools/0000000d-000d-000d-000d-00000000037a" id="0000000d-000d-000d-000d-00000000037a"/>
-    <quota_mode>disabled</quota_mode>
+  <name>Default</name>
+  <description>The default Data Center</description>
+  <link href="/api/datacenters/00000001-0001-0001-0001-0000000000f1/storagedomains" rel="storagedomains"/>
+  <link href="/api/datacenters/00000001-0001-0001-0001-0000000000f1/clusters" rel="clusters"/>
+  <link href="/api/datacenters/00000001-0001-0001-0001-0000000000f1/networks" rel="networks"/>
+  <link href="/api/datacenters/00000001-0001-0001-0001-0000000000f1/permissions" rel="permissions"/>
+  <link href="/api/datacenters/00000001-0001-0001-0001-0000000000f1/quotas" rel="quotas"/>
+  <link href="/api/datacenters/00000001-0001-0001-0001-0000000000f1/qoss" rel="qoss"/>
+  <link href="/api/datacenters/00000001-0001-0001-0001-0000000000f1/iscsibonds" rel="iscsibonds"/>
+  <local>false</local>
+  <storage_format>v3</storage_format>
+  <version major="3" minor="6"/>
+  <supported_versions>
+      <version major="3" minor="6"/>
```

Comment:
small formatting inconsistency. Everywhere we have 2 spaces and here and in line 132 we have 4.